README.md

```diff
@@ -5,6 +5,13 @@ to use our drone, simply add to your `Cargo.toml`, under `[dependencies]`:
 ``` rust
 null-pointer-drone = { git = "https://github.com/The-Null-Pointer-Patrol/null-pointer-drone.git"}
 ```
+Once you done that you can declare :
+``` rust 
+use null_pointer_drone::MyDrone; 
+```
+# After-sales service
+if we have any problem with our drone you can still open an issue on our github at [this link](https://github.com/The-Null-Pointer-Patrol/null-pointer-drone/issues)
```

Comment:
if "you" have/ if you encounter, and i would remove "still"

Reply:
Yeah, maybe my English is not so good. I understand that